tic_tac_toe/game.py

```diff
@@ -24,7 +26,10 @@ def _position_is_valid(position):
 
     Returns True if given position is valid, False otherwise.
     """
-    pass
+    if not isinstance(position, tuple) or len(position) != 2 or not isinstance(position[0], int) or not isinstance(position[1], int):
+        return False
+        
+    return 0 <= position[0] <= 2 and 0 <= position[1] <= 2
```

Comment:
Checking that the elements in the position tuple are ints is a very good idea, this is something we did not think to do. If I were to nit-pick I suppose the strings '0', '1', '2'... could also be allowed also.